DataManagementSystem/Agent/RequestOperations/ReplicateAndRegister.py

```diff
@@ -56,6 +56,15 @@ def filterReplicas( opFile, logger = None, dataManager = None ):
     return S_ERROR( failed )
 
   replicas = replicas["Successful"].get( opFile.LFN, {} )
+  if not replicas:
```

Comment:
If I understand well, if we don't have any active replicas, you get all the replicas, correct ? But it is not really useful if we want to replicate the replica and the SE is unavailable